product/infrastructure/fake/searchservice.go

```diff
@@ -17,27 +19,39 @@ type (
 		productService         *ProductService
 		liveSearchJSON         string
 		categoryFacetItemsJSON string
+		sortConfig             []sortConfig
 	}
 	liveSearchData struct {
 		Marketplacecodes []string                  `json:"marketplacecodes"`
 		Sugestions       []searchDomain.Suggestion `json:"sugestions"`
 		Promotions       []searchDomain.Promotion  `json:"promotions"`
 		Actions          []searchDomain.Action     `json:"actions"`
 	}
+
+	// sortConfig contains sorting configuration for search results
+	sortConfig struct {
+		Key   string
+		Label string
+		Asc   string
+		Desc  string
+	}
 )
 
 // Inject dependencies
 func (s *SearchService) Inject(
 	productService *ProductService,
 	cfg *struct {
-		LiveSearchJSON         string `inject:"config:commerce.product.fakeservice.jsonTestDataLiveSearch,optional"`
-		CategoryFacetItemsJSON string `inject:"config:commerce.product.fakeservice.jsonTestDataCategoryFacetItems,optional"`
+		LiveSearchJSON         string       `inject:"config:commerce.product.fakeservice.jsonTestDataLiveSearch,optional"`
+		CategoryFacetItemsJSON string       `inject:"config:commerce.product.fakeservice.jsonTestDataCategoryFacetItems,optional"`
+		SortConfig             config.Slice `inject:"config:commerce.product.fakeservice.sorting"`
```

Comment:
we should put something to the cueconfig of the module

Reply:
done